docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -41,43 +32,20 @@ Within every game (XML) file, inside the "games" folder inside the main folder o
 - `="Battleships repair at beginning of round"` >> `="Units Repair Hits Start Turn"`
 - `="Naval Bombard Casualties Return Fire Restricted"` >> `="Naval Bombard Casualties Return Fire"`
 - `="SBR Affects Unit Production"` >> `="Damage From Bombing Done To Units Instead Of Territories"`
-
-### Changes of game (XML) file special code for the above items:
-
-_(these are cases of advanced coding, so you probably don't have these within the game file)_
-
-Within every game file (XML), inside the "games" folder inside the main folder of the map, change:
-
 - `="conditionType" count="XOR"` >> `="conditionType" count="1"`
 - `="conditionType" count="-reset-XOR"` >> `="conditionType" count="-reset-1"`
-
-### Changes of already deprecated game (XML) file items:
-
-_(these items were already deprecated, so you shouldn't have had these within the game file already)_
-
-Within every game (XML) file, inside the "games" folder inside the main folder of the map, change:
-
-- `="isTwoHit" value="true"` >> `="hitPoints" value="2"`
-- `="isTwoHit" value="false"` >> `="hitPoints" value="1"`
-- `="occupiedTerrOf"` >> `="originalOwner"`
-- `="victoryCity" value="true"` >> `="victoryCity" value="1"`
```

Comment:
The new document is completely missing this item. I believe it has been removed by mistake, thus needs to be added back.

Reply:
Please check again, this is present in the latest:

> ="victoryCity" count="true" >> ="victoryCity" count="1"
="victoryCity" count="-reset-true" >> ="victoryCity" count="-reset-1"
="victoryCity" count="false" >> ="victoryCity" count="0"
="victoryCity" count="-reset-false" >> ="victoryCity" count="-reset-0"

Reply:
There is no "`="victoryCity" value="true"` >> `="victoryCity" value="1"`" in what you are quoting.